src/runtime-rs/crates/hypervisor/src/qemu/inner.rs

```diff
@@ -124,6 +125,22 @@ impl QemuInner {
                         network.config.guest_mac.clone().unwrap(),
                     )?;
                 }
+                DeviceType::Protection(prot_dev) => match &prot_dev.config {
+                    ProtectionDeviceConfig::SevSnp(sev_snp_cfg) => {
+                        if sev_snp_cfg.is_snp {
+                            cmdline.add_sev_snp_protection_device(
+                                sev_snp_cfg.cbitpos,
+                                &sev_snp_cfg.firmware,
+                                &sev_snp_cfg.certs_path,
```

Comment:
what happens if you add a `certs_path` on a non-SNP SEV?

Reply:
This code is only executed if the `is_snp` condition on line 130 above is true and we know that it's SEV-SNP so we call `add_sev_snp_protection_device()`.

For a non-SNP SEV the  `is_snp` will presumably be false and the other branch (below) will execute that calls `add_sev_protection_device()`.

So if `certs_path` is set for a non-SNP SEV device I guess we can say it will be ignored.

Also, `certs_path` is deprecated anyway IIUC and will be removed in a subsequent PR once this one is merged.